CPT331.iOS/CPT331.iOS/Event.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Event.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 22/09/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+struct Event {
```

Comment:
I did a bit of googling on the Swift docs
https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/ClassesAndStructures.html

The struct is a value type like most other languages. I would think a class would be nicer, particularly as it contains string data of an undefined length. Class types should be faster to pass as an argument (by reference).